projects/magento/pageObjects/checkout/SuccessfulCheckout.page.js

```diff
@@ -12,6 +12,7 @@ export class SuccessfulCheckoutPage extends BasePage {
     this.voucherCodeContainer = page.locator(
       ".adyen-checkout__voucher-result"
     );
+    this.orderNumber = page.locator(".checkout-success p span");
```

Comment:
How about this?

```suggestion
    get orderNumber() {
      return page.locator(".checkout-success p span").innerText();
    }  
```

Reply:
Thanks for the input, I made a slightly different implementation